samples/non-python-preprocess/devops_pipelines/dependency-update-template.yml

```diff
@@ -0,0 +1,10 @@
+# Pipeline template to update the custom docker image with latest dependencies
+steps:
+- script: |   
+   conda env update -f local_development/dev_dependencies.yml -n ci
+  workingDirectory: ${{ parameters.workingDirectory }}
+  displayName: 'Download pip packages with latest dependency file'
+- script: |   
```

Comment:
@h2floh Do you think can I leave this code to check required package installed or is it better to remove?

Reply:
Like it, will help debugging if the pipeline fails!